Project: Library

gscarv13 opened the pull request:
#### This pull request contains the following:
- The user can create new `Book` objects filling the form
- All books created are saved using `Local Storage` 
- UI built using the `Bulma` CSS framework

gscarv13 committed:
Add initial files

gscarv13 committed:
Add basic form

gscarv13 committed:
Add book constructor, add addBookToLibrary function

gscarv13 committed:
Add Github Actions, Add .gitignore file

gscarv13 committed:
Update arrow function, Add openBookForm, Add even listeners

gscarv13 committed:
Add hide-form class, Add html table skeleton

gscarv13 committed:
Add display function

gscarv13 committed:
Add delete function

gscarv13 committed:
Add icons to the markup

gscarv13 committed:
Add ngrok

gscarv13 committed:
Add sample books, Implement local storage

gscarv13 committed:
Add functions to update local storage entry, Update delete function

gscarv13 committed:
Improve UI

gscarv13 committed:
Add simple validation before saving book

gscarv13 committed:
Fix stylelint error

gscarv13 committed:
Add README

gscarv13 committed:
Fix description

rindrajosia reviewed (changes requested):
Changes Required :red_circle:
To Highlight: :angel:

- Good job using local storage :1st_place_medal: 
- Books are displayed :1st_place_medal: 
- User can add book :+1: 
- User can remove book :100: 
- User can change book status :+1: 

To improve :factory_worker: [REQUIRED]:

- [x] I suggest that you use tools like [grammarly](https://app.grammarly.com/) in order to help you with your readme file as it is the first thing user checks.
- [x] It will be nice if you add screenshot in your readme.

Nice to have:

- [ ]  The form used to create a book should have validations, user should not be able to create the same  book twice.

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

gscarv13 committed:
Update readme, Centered book table

gscarv13 committed:
Replace innerHTML with textContent

gscarv13 committed:
Add screenshot

gscarv13 commented:
## Thank you for your review @rindrajosia 

The following changes have been made:
- Fixed typo on the readme
- Added screenshot
- Centered UI elements

oloomoses reviewed (approved):
# Congratulations, Project Approved. :muscle: :tada: :clap: 
**Hello @gscarv13, 
Well done implementing the suggested changes from the previous reviews.
You have passed all the requirements for this project :muscle: 
Feel free to merge and close this pull request.**

**I wish you the best in your next project and see you at the top :rocket:** 
#### Happy Coding
_Slack: Oloo Moses_
